### PR TITLE
Add `allowCamelCase` option

### LINKED ISSUE #33: camelCase option
Are you open to exposing an option `camelCase` (or something similar) which would bypass the `replace` and `toLowerCase` code found here: https://github.com/sindresorhus/dargs/blob/master/index.js#L14 ?

I'd be happy to submit a PR...

### PR DESCRIPTION
**Pull request by jaridmargolin**
Close #33 by adding `allowCamelCase` option.

### CODE REVIEW

**Comment by jaridmargolin** [comment]
An additional note. I chose to use `allowCamelCase` rather than `camelCase` as I thought it would be misleading to use `camelCase` and not convert hyphenated args to camelcase. The current option is used only to bypass the existing conversion.

File: readme.md [hunk 1]
```diff
@@ -138,6 +138,23 @@ Default: `false`
...
+```
+console.log(dargs({ 'fooBar': 'baz' }));
+/*
+['--foo-bar', 'baz']
+*/
+console.log(dargs({ 'fooBar': 'baz' }, { allowCamelCase: true }));
+/*
+['--fooBar', 'baz']
+*/
+```
```
Comment (kevva) [inline readme.md]: Write it as this instead:

```js
console.log(dargs({fooBar: 'baz}));
//=> ['--foo-bar', 'baz']

console.log(dargs({fooBar: 'baz}, {allowCamelCase: true}));
//=> ['--fooBar', 'baz']
```

Comment (jaridmargolin) [inline readme.md]: 👍 done

File: readme.md [hunk 2]
```diff
@@ -138,6 +138,19 @@ Default: `false`
...
+
+Type: `boolean`<br>
+Default: `false`
+
+By default dargs will convert camelcase arguments to hyphenated arguments. Setting this to true will bypass the conversion process. For example:
+
+```
+console.log(dargs({ 'fooBar': 'baz' }));
+//=> ['--foo-bar', 'baz']
+console.log(dargs({ 'fooBar': 'baz' }, { allowCamelCase: true }));
```
Comment (kevva) [inline readme.md]: Empty line here between the examples :).

Comment (jaridmargolin) [inline readme.md]: 👍 :)

File: readme.md [hunk 3]
```diff
@@ -138,6 +138,20 @@ Default: `false`
...
+
+By default dargs will convert camelcase arguments to hyphenated arguments. Setting this to true will bypass the conversion process. For example:
+
+```
+console.log(dargs({ 'fooBar': 'baz' }));
+//=> ['--foo-bar', 'baz']
+
+console.log(dargs({ 'fooBar': 'baz' }, { allowCamelCase: true }));
+//=> [--fooBar', 'baz']
+```
```
Comment (jaridmargolin) [inline readme.md]: Some impressively terrible attention to detail on my part. Sorry about that. Should be good now?

**Pull request merged**